### DISCUSSION TITLE
Disable grey text in completion

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [LazyVim/LazyVim#4576] [Q&A]
Hi, 

What is the name of the code completion functionality below?
Whenever I go through the list of suggestions it aligns the existing code to the text in focus in the list and it is very annoying for me.
<img width="605" alt="Screenshot 2024-10-12 at 07 52 48" src="https://github.com/user-attachments/assets/054debe9-2d89-4f04-9114-373943f1c133">

**Comment by dpetka2001** [comment]
I believe the OP is talking about the `ghost_text` feature of nvim-cmp, not the opacity of the nvim-cmp auto-completion popup menu. To disable that you can create a file `~/.config/nvim/lua/plugins/cmp.lua` with contents
```lua
return {
  {
    "hrsh7th/nvim-cmp",
    opts = {
      experimental = {
        ghost_text = false,
      },
    },
  },
}
```
